build/dsc/resource.rb

```diff
@@ -110,6 +110,10 @@ def ensure_value
           'present'
         elsif ensure_property.values.any? { |v| v.casecmp('enable') == 0 }
           'enable'
+        elsif ensure_property.values.any? { |v| v.casecmp('published') == 0 }
+          'published'
```

Comment:
This is required because https://github.com/PowerShell/xSCSMA/blob/dev/DSCResources/MSFT_xRunbookDirectory/MSFT_xRunbookDirectory.schema.mof#L4 uses non-standard values of Ensure